src/claims/models/investor-uniqueness.model.ts

```diff
@@ -0,0 +1,14 @@
+import { ApiProperty, OmitType, PartialType } from '@nestjs/swagger';
+
+import { ClaimModel } from '~/claims/models/claim.model';
+import { InvestorUniquenessClaimModel } from '~/claims/models/investor-uniqueness-claim.model';
+import { FromEntity } from '~/common/decorators/transformation';
+
+export class InvestorUniquenessModel extends PartialType(OmitType(ClaimModel, ['claim'] as const)) {
+  @ApiProperty({
+    description: 'Details of the Claim containing type and scope',
+    type: InvestorUniquenessClaimModel,
+  })
+  @FromEntity()
+  readonly claim: InvestorUniquenessClaimModel;
+}
```

Comment:
We can get rid of this class and directly use ClaimModel<InvestorUniquenessClaimModel>, only catch would be that to properly handle the swagger schema. We would probably have to create a custom decorator to handle this I suppose. Thoughts @polymath-eric ?